src/test/kotlin/apply/domain/applicationform/ApplicationFormTest.kt

```diff
@@ -0,0 +1,94 @@
+package apply.domain.applicationform
+
+import apply.domain.recruitmentitem.Answer
+import apply.domain.recruitmentitem.Answers
+import org.assertj.core.api.Assertions.assertThat
+import org.assertj.core.api.Assertions.assertThatIllegalArgumentException
+import org.junit.jupiter.api.BeforeEach
+import org.junit.jupiter.api.Test
+import org.junit.jupiter.api.assertAll
+
+internal class ApplicationFormTest {
+    private lateinit var applicationForm: ApplicationForm
+
+    @BeforeEach
+    internal fun setUp() {
+        applicationForm = ApplicationForm(
+            applicantId = 1L,
+            recruitmentId = 1L,
+            referenceUrl = "http://example.com",
+            answers = Answers(
+                mutableListOf(
+                    Answer(
+                        "스타트업을 하고 싶습니다.",
+                        1L
+                    ),
+                    Answer(
+                        "책임감",
+                        2L
+                    )
```

Comment:
아래의 코드로 개선해 보면 어떨까요?

```suggestion
                    Answer("스타트업을 하고 싶습니다.", 1L),
                    Answer("책임감", 2L)
```